.github/workflows/test-runner-on-perf.yml

```
@@ -80,70 +89,70 @@ jobs:
+          branch: "version-update-${{ steps.read_property.outputs.LATEST_SEMVER }}"
           body: |
-            Update versions in perf env to reflect image tag ${{ steps.read_property.outputs.LATEST_GITHASH }}.
+            Update versions in perf env to reflect image tag ${{ steps.read_property.outputs.LATEST_SEMVER }}.
             *Note: This PR was opened by the [test-runner-perf GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
           labels: "broadbot,datarepo,automerge"
       - name: "Merge datarepo-helm-definition pull request"
+        if: ${{ steps.read_property.outputs.LATEST_SEMVER != steps.read_property.outputs.CURRENT_SEMVER }}
         run: |
           cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-helm-definitions
           git checkout master
           git config pull.rebase false
-          if git rev-parse --verify origin/version-update-${{ steps.read_property.outputs.LATEST_GITHASH }}; then
-            git merge origin/version-update-${{ steps.read_property.outputs.LATEST_GITHASH }}
+          if git rev-parse --verify origin/version-update-${{ steps.read_property.outputs.LATEST_SEMVER }}; then
+            git merge origin/version-update-${{ steps.read_property.outputs.LATEST_SEMVER }}
             git push
-            git push origin --delete version-update-${{ steps.read_property.outputs.LATEST_GITHASH }}
+            git push origin --delete version-update-${{ steps.read_property.outputs.LATEST_SEMVER }}
           else
-            echo "Branch version-update-${{ steps.read_property.outputs.LATEST_GITHASH }} not found."
+            echo "Branch version-update-${{ steps.read_property.outputs.LATEST_SEMVER }} not found."
           fi
           cd ${GITHUB_WORKSPACE}/${workingDir}
       - name: "Install Helmfile"
+        if: ${{ steps.read_property.outputs.LATEST_SEMVER != steps.read_property.outputs.CURRENT_SEMVER }}
         uses: broadinstitute/setup-helmfile@v0.5.0 #Forked from mamezou-tech/setup-helmfile
       - name: "Use helmfile to apply image tag update to Perf yaml"
+        if: ${{ steps.read_property.outputs.LATEST_SEMVER != steps.read_property.outputs.CURRENT_SEMVER }}
         run: |
           helmfile --version
           cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-helm-definitions/perf
           helmfile apply
           cd ${GITHUB_WORKSPACE}/${workingDir}
       - name: "Wait for Perf Cluster to come back up with correct version"
+        if: ${{ steps.read_property.outputs.LATEST_SEMVER != steps.read_property.outputs.CURRENT_SEMVER }}
         run: |
-          PERF_VERSION=$(curl -s -X GET "https://jade-perf.datarepo-perf.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.gitHash')
+          PERF_VERSION=$(curl -s -X GET "https://jade-perf.datarepo-perf.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer' | sed 's/-SNAPSHOT//')
```

It's not a vital change, but `jq` can do many `sed`-like operations already
```suggestion
          PERF_VERSION=$(curl -s -X GET "https://jade-perf.datarepo-perf.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
```

---

The true JQ wizard! 🧙

---

Thanks, @pshapiro4broad!